20231114_map-based-noise/README.md

```diff
@@ -0,0 +1,32 @@
+LiteBIRD map based simulations of noise
+=======================================
+
+## Summary
+500 realizations of white noise per frequency channel for the LiteBIRD satellite.
+unit = `K_CMB`, nside = 512, coord = Galactic
+
+## Description of simulation
+Maps were generated from white noise covariances provided in the first release of the LiteBIRD end-to-end
```

Comment:
white noise covariances should also be part of the release

Reply:
@toshiyan and this

Reply:
the covariance is also saved now.